<ui version="4.0" >
 <class>StringListEditor</class>
 <widget class="QWidget" name="StringListEditor" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>554</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>AdressList</string>
  </property>
  <layout class="QGridLayout" >
   <property name="leftMargin" >
    <number>0</number>
   </property>
   <property name="topMargin" >
    <number>0</number>
   </property>
   <property name="rightMargin" >
    <number>0</number>
   </property>
   <property name="bottomMargin" >
    <number>0</number>
   </property>
   <property name="horizontalSpacing" >
    <number>6</number>
   </property>
   <property name="verticalSpacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" colspan="2" >
    <widget class="QLabel" name="lblTitle" >
     <property name="text" >
      <string>TITLE  :</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QLineEdit" name="editString" />
   </item>
   <item row="1" column="1" >
    <widget class="QToolButton" name="btnAdd" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="icon" >
      <iconset resource="dyprosium.qrc" >:/images/Resources/add.png</iconset>
     </property>
     <property name="toolButtonStyle" >
      <enum>Qt::ToolButtonIconOnly</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QListWidget" name="listString" />
   </item>
   <item row="2" column="1" >
    <layout class="QVBoxLayout" >
     <item>
      <widget class="QToolButton" name="btnRemove" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="dyprosium.qrc" >:/images/Resources/delete.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnUp" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="dyprosium.qrc" >:/images/Resources/arrow_up.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="btnDown" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="dyprosium.qrc" >:/images/Resources/arrow_down.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources>
  <include location="dyprosium.qrc" />
 </resources>
 <connections/>
</ui>
